doc/modules/calibration.rst

```diff
@@ -208,3 +208,124 @@ a similar decrease in log-loss.
+all decision thresholds calculated on the hold-out parts of each cross
+validation iteration. Finally the model is trained on all the provided data.
+When using ``cv = 'prefit'`` you need to make sure to use a hold-out part of
+your data for calibration.
+
+The strategies, controlled by the parameter ``strategy``, for finding
+appropriate decision thresholds are based either on precision recall estimates
+or true positive and true negative rates. Specifically:
+
+.. currentmodule:: sklearn.metrics
+
+* ``f_beta``
+   selects a decision threshold that maximizes the :func:`fbeta_score`. The
+   value of beta is specified by the parameter ``beta``. The ``beta`` parameter
+   determines the weight of precision. When ``beta = 1`` both precision recall
+   get the same weight therefore the maximization target in this case is the
+   :func:`f1_score`. if ``beta < 1`` more weight is given to precision whereas
+   if ``beta > 1`` more weight is given to recall.
+
+* ``roc``
+   selects the decision threshold for the point on the :func:`roc_curve` that
+   is closest to the ideal corner (0, 1)
+
+* ``max_tpr``
+   selects the decision threshold for the point that yields the highest true
+   positive rate while maintaining a minimum true negative rate, specified by
+   the parameter ``threshold``
+
+* ``max_tnr``
+   selects the decision threshold for the point that yields the highest true
+   negative rate while maintaining a minimum true positive rate, specified by
+   the parameter ``threshold``
+
+Here is a simple usage example::
+
+   >>> from sklearn.calibration import CutoffClassifier
+   >>> from sklearn.datasets import load_breast_cancer
+   >>> from sklearn.naive_bayes import GaussianNB
+   >>> from sklearn.metrics import precision_score
+   >>> from sklearn.model_selection import train_test_split
+
+   >>> X, y = load_breast_cancer(return_X_y=True)
+   >>> X_train, X_test, y_train, y_test = train_test_split(
+   ...     X, y, train_size=0.6, random_state=42)
+   >>> clf = CutoffClassifier(GaussianNB(), strategy='f_beta', beta=0.6,
+   ...                        cv=3).fit(X_train, y_train)
+   >>> y_pred = clf.predict(X_test)
+   >>> precision_score(y_test, y_pred)                   # doctest: +ELLIPSIS
+   0.959...
+
+.. topic:: Examples:
+
+ * :ref:`sphx_glr_auto_examples_calibration_plot_decision_threshold_calibration.py`: Decision
+   threshold calibration on the breast cancer dataset
+
+.. currentmodule:: sklearn.calibration
+
+The following image shows the results of using the :class:`CutoffClassifier`
+for finding a decision threshold for a :class:`LogisticRegression` classifier
+and an :class:`AdaBoostClassifier` for two use cases.
+
+.. figure:: ../auto_examples/calibration/images/sphx_glr_plot_decision_threshold_calibration_001.png
+   :target: ../auto_examples/calibration/plot_decision_threshold_calibration.html
+   :align: center
+
+In the first case we want to increase the overall accuracy of the classifier on
```

Comment:
I think if this is described inside the example file, we do not need to repeat it here.